modules/internal/Bytes.chpl

```diff
@@ -174,21 +174,24 @@ module Bytes {
 
   /*
      Creates a new :mod:`bytes <Bytes>` which borrows the memory allocated for a
-     `c_ptr(uint(8))`. If the buffer is freed before the :mod:`bytes <Bytes>`
-     returned from this function, accessing it is undefined behavior.
+     `c_ptr`. If the buffer is freed before the :mod:`bytes <Bytes>` returned
+     from this function, accessing it is undefined behavior.
 
      :arg s: Buffer to borrow
-     :type s: `bufferType` (i.e. `c_ptr(uint(8))`)
+     :type x: `c_ptr` to `uint(8)` or `c_char`
```

Comment:
Would you be open to writing both types out?

```suggestion
     :type x: `c_ptr(uint(8))` or `c_ptr(c_char)`
```

Reply:
rationale: I think it's easier read quickly and leaves less up for interpretation.

This applies in other docs as well.

Reply:
Yeah, that makes sense. I'll make that change